proto/config/netinst.proto

```diff
@@ -75,6 +75,13 @@ message NetworkInstanceLispConfig {
   bool experimental = 20;
 }
 
+message IPRoute {
+  // Destination network address in the CIDR format: <IP-address>/<prefix-length>
+  string destination_network = 1;
+  // Gateway IP address
+  string gateway = 2;
```

Comment:
Are we planning to allow some semantics around this being empty/unspecified?
If so we should have a comment about the semantics of an empty string or 0.0.0.0.
If not we should say that this must be set to a valid IP address and can not be all-zeros.

Reply:
Done, comment extended (empty IP not allowed).